From ea25d88d77004b87ed2a90303fdee7f447d6dfa6 Mon Sep 17 00:00:00 2001
From: Daniel Kurtz <djkurtz@chromium.org>
Date: Wed, 28 Aug 2019 00:52:35 -0600
Subject: [PATCH] Fixes for OpenSSL 1.1 build

Small fixes to fix building against openssl-1.1:
1) Remove dso.h include; this file has been removed from openssl-1.1;
 further nothing in the code uses this reference anything from this
 file.
2) Include rand.h for RAND_SSLeay declaration.
---
 src/hw_pkcs11.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hw_pkcs11.c b/src/hw_pkcs11.c
index 24806ff..85e46db 100644
--- a/src/hw_pkcs11.c
+++ b/src/hw_pkcs11.c
@@ -65,7 +65,7 @@
 #include <openssl/crypto.h>
 #include <openssl/objects.h>
 #include <openssl/engine.h>
-#include <openssl/dso.h>
+#include <openssl/rand.h>
 #ifndef ENGINE_CMD_BASE
 #error did not get engine.h
 #endif
-- 
2.23.0.187.g17f5b7556c-goog

